code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -517,10 +513,12 @@
 		var/mob/living/simple_animal/hostile/H = M
 		H.AIStatus = initial(H.AIStatus)
 
-
 /obj/effect/timestop/wizard
 	duration = 100
 
+/obj/effect/timestop/wizard/New()
+	..()
+	timestop()
```

Comment:
yes, which is why you call something like `created()` instead and override THAT to not timestop on the sepia child